src/System.Private.Interop/src/System.Private.Interop.Mono.csproj

```diff
@@ -1,39 +1,107 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <Import Project="System.Private.Interop.CoreCLR.csproj" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <AssemblyName>System.Private.Interop</AssemblyName>
     <TargetName>$(AssemblyName)</TargetName>
-    <DefineConstants>$(DefineConstants);ENABLE_MIN_WINRT</DefineConstants>
+    <OutputType>Library</OutputType>
```

Comment:
This is going to drop the binaries under sdk\mono. I would like everything under sdk to be CoreRT runtime/toolchain specific. Could you please drop these files under e.g. `interop`.

Reply:
do you mean interop folder and sdk folder are under same parent folder? like
sdk
interop

Reply:
Right

Reply:
Fixed!